style.css

```diff
@@ -0,0 +1,720 @@
+  width: 80%;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: 10%;
+}
+
+.about-footer-image {
+  width: 80%;
+  padding: 8px;
+}
+
+.desk-about-sum-display {
+  display: none;
+}
+
+@media screen and (min-device-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+
+  .desk-nav {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .fixed-top {
+    display: flex;
+    justify-content: end;
+    background: #272a31;
+  }
+
+  .logo-image {
+    height: 100%;
+  }
+
+  .logo-div {
+    margin-left: 13%;
+  }
+
+  .about-link {
+    font-family: 'Lato', sans-serif;
+    text-decoration: none;
+    color: #272a31;
+  }
+
+  .fixed-ul {
+    display: flex;
+    color: #fff;
+    list-style: none;
+    width: 22%;
+    margin-right: 9%;
+    justify-content: space-between;
+  }
+
+  .main-nav {
+    background: #fff;
+  }
+
+  .desk-nav-bg {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .menu-items {
+    display: flex;
+    width: 50%;
+    margin-right: 10%;
+    justify-content: space-between;
+  }
+
+  .menu-items ul {
+    display: flex;
+    list-style: none;
+    justify-content: space-between;
+    width: 75%;
+  }
+
+  .menu-button {
+    color: #ec5242;
+    height: 75%;
+    border: 5px solid #ec5242;
+    margin-top: 5px;
+    background: #fff;
+  }
+
+  .intro-texts {
+    margin-top: 100px;
+    width: 60%;
+    margin-right: 15%;
+  }
+
+  .text-excl {
+    font-size: 3em;
+    margin-bottom: 0;
+  }
+
+  .top-header {
+    font-size: 5em;
+    margin-top: 0;
+    margin-bottom: 15px;
+  }
+
+  .foreword {
+    width: 75%;
+    font-size: 1.2em;
+    margin-top: 0;
+    padding: 15px;
+  }
+
+  .event-date {
+    font-size: 2em;
+  }
+
+  .last-text {
+    font-size: 1.3em;
+    margin-bottom: 200px;
+  }
+
+  .desk-prog {
+    display: flex;
+    width: 80%;
+    margin-left: 10%;
+  }
+
+  .program-card {
+    display: flex;
+    flex-direction: column;
+    margin-left: 5px;
+  }
+
+  .card-image {
+    width: 25%;
+    margin-left: 25%;
+    margin-bottom: 0;
+  }
+
+  .button-space {
+    display: none;
+  }
+
+  .title-text {
+    font-size: 2em;
+    color: #fff;
+  }
+
+  .desk-prog-info {
+    color: #fff;
+    font-size: 1.5em;
+    display: flex;
+    justify-content: center;
+    text-decoration: underline;
+    margin-top: 20px;
+    margin-bottom: 100px;
+  }
+
+  .horizontal-line {
+    width: 3%;
+  }
+
+  .card-info {
+    margin-top: 0;
+    padding: 0 5%;
+  }
+
+  .card-title {
+    font-size: 1.5em;
+    margin-bottom: 10px;
+  }
+
+  .speaker-title {
+    color: #000;
+  }
+
+  .speaker-card {
+    display: none;
+  }
+
+  .more-speakers {
+    display: none;
+  }
+
+  .desk-speaks {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-row-gap: 80px;
+    width: 80%;
+    margin-left: 10%;
+    margin-bottom: 100px;
+  }
+
+  .desk-card {
+    display: flex;
+  }
+
+  .desk-card-img {
+    width: 25%;
+    height: 100%;
+    padding: 8px;
+  }
```

Comment:
- [ ] I think it would be better if you increase a little bit the width of the image tp be more visible 🙃 😉